dbms/src/Storages/DeltaMerge/BitmapFilter/BitmapFilter.cpp

```diff
@@ -97,6 +97,42 @@ void BitmapFilter::rangeAnd(IColumn::Filter & f, UInt32 start, UInt32 limit) con
     }
 }
 
+void BitmapFilter::logicalOr(const BitmapFilter & other)
+{
+    RUNTIME_CHECK(filter.size() == other.filter.size());
+    if (all_match)
+    {
+        return;
+    }
+    if (other.all_match)
+    {
+        filter.assign(filter.size(), true);
```

Comment:
```suggestion
        filter.assign(filter.size(), static_cast<UInt8>(true));
```

Otherwise it will match the `void assign(It1 from_begin, It2 from_end)`